tests/pricing_models/hyperdrive/hyperdrive_generate_expected_results.py

```diff
@@ -0,0 +1,256 @@
+# pylint: skip-file
```

Comment:
I'm against merging unlinted files into the repo. It's fine to keep around a useful tool, but it needs to meet the repo's standards.

Reply:
This file needs to be cleaned up before I'll approve.

I also just want to make sure that this is not replacing having your eyes on the inputs and outputs to make sure they are reasonable. I used a calculator (`python -i`) when I was writing the original pricing model tests, so it's totally fine to use another tool; however, we just want to make sure that we're carefully considering what we're putting into our test cases as we go along. I'm not suggesting that you aren't doing this, but I just think it's worth calling out.

Reply:
Yeah totally.  I'm not actually running anything through a calculator, but I'm visually inspecting things to make sure they look approximately sane.  I think a lot of the value for these tests come from the fact that the output is generated by a function that has all the math inlined, so we can inspect that the implementation code is doing what we want it to do.